waspc/src/Wasp/Generator/DbGenerator.hs

```diff
@@ -156,12 +115,8 @@ genMigrationsDir spec =
   where
     genProjectMigrationsDir = dbRootDirInProjectRootDir </> dbMigrationsDirInDbRootDir
 
-writeDbSchemaChecksumToFile :: Path' Abs (Dir ProjectRootDir) -> IO ()
-writeDbSchemaChecksumToFile genProjectRootDir = do
-  dbSchemaExists <- doesFileExist dbSchemaFp
-  when dbSchemaExists $ do
-    checksum <- hexToString <$> checksumFromFilePath dbSchemaFp
-    writeFile dbSchemaChecksumFp checksum
-  where
-    dbSchemaFp = SP.fromAbsFile $ genProjectRootDir </> dbSchemaFileInProjectRootDir
-    dbSchemaChecksumFp = SP.fromAbsFile $ genProjectRootDir </> dbSchemaChecksumFileInProjectRootDir
+-- | NOTE(shayne): We cannot check for the difference between the prisma schema and checksum file here, as we may need
+-- to regnerate the client after it was "dirtied" by a change that made the checksums differ and then was undone.
+-- TODO(shayne): Any other heuristics we can use to do this less often?
```

Comment:
The reason we can't diff checksum is for example:
- Change entity column, checksum diffs, we regenerate client
- Undo that change, checksum same, we do not regenerate client but it is now still the old/last/wrong change version

Reply:
Therefore, I thought it was safest to just always run for now (until some better heuristic is added).

Reply:
We could also add another prisma checksum file, one for last migrate, and one for last generate, and then we can check that one.

Reply:
Yeah, I probably should have just pulled the trigger on that optimization myself to begin with. I will add a checksum for `generate` so we don't have to call it all the time and keep compilation speedy in the general case 👍🏻

Reply:
Yes that is it, two checksum files!

Reply:
Added second checksum to speed it up! 👍🏻